docs/user/custom-resources/apirule/04-10-apirule-custom-resource.md

```diff
@@ -85,53 +29,248 @@ spec:
+| **rules** <br /> [Rule](#rule) array | Defines an ordered list of access rules. Each rule is an atomic configuration that<br />defines how to access a specific HTTP path. A rule consists of a path<br />pattern, one or more allowed HTTP methods, exactly one access strategy (**jwt**, **extAuth**,<br />or **noAuth**), and other optional configuration fields. | MinItems: 1 <br /> |
+| **timeout** <br /> [Timeout](#timeout) | Specifies the timeout for HTTP requests in seconds for all rules.<br />You can override the value for each rule. If no timeout is specified, the default timeout of 180 seconds applies. | Maximum: 3900 <br />Minimum: 1 <br /> |
+
+### APIRuleStatus
+
+Describes the observed status of the APIRule.
+
+Appears in:
+- [APIRule](#apirule)
+
+| Field | Description | Validation |
+| --- | --- | --- |
+| **lastProcessedTime** <br /> [Time](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.33/#time-v1-meta) | Represents the last time the APIRule status was processed. | None |
+| **state** <br /> [State](#state) | Defines the reconciliation state of the APIRule.<br />The possible states are `Ready`, `Warning`, or `Error`. | Enum: [Processing Deleting Ready Error Warning] <br />Required <br /> |
+| **description** <br /> string | Contains the description of the APIRule's status. | None |
+
+### CorsPolicy
+
+Allows configuring CORS headers sent with the response. If **corsPolicy** is not defined,
+the CORS headers are removed from the response.
+
+Appears in:
+- [APIRuleSpec](#apirulespec)
+
+| Field | Description | Validation |
+| --- | --- | --- |
+| **allowHeaders** <br /> string array | Indicates whether credentials are allowed in the **Access-Control-Allow-Credentials** CORS header. | None |
+| **allowMethods** <br /> string array | Lists headers allowed with the **Access-Control-Allow-Headers** CORS header. | None |
+| **allowOrigins** <br /> [StringMatch](#stringmatch) | Lists headers allowed with the **Access-Control-Allow-Methods** CORS header. | None |
+| **allowCredentials** <br /> boolean | Lists origins allowed with the **Access-Control-Allow-Origins** CORS header. | None |
+| **exposeHeaders** <br /> string array | Lists headers allowed with the **Access-Control-Expose-Headers** CORS header. | None |
+| **maxAge** <br /> integer | Specifies the maximum age of CORS policy cache. The value is provided in the **Access-Control-Max-Age** CORS header. | Minimum: 1 <br /> |
+
+### ExtAuth
+
+**ExtAuth** contains configuration for paths that use external authorization.
+
+Appears in:
+- [Rule](#rule)
+
+| Field | Description | Validation |
+| --- | --- | --- |
+| **authorizers** <br /> string array | Specifies the name of the external authorization handler. | MinItems: 1 <br /> |
+| **restrictions** <br /> [JwtConfig](#jwtconfig) | Specifies JWT configuration for the external authorization handler. | None |
+
+### Host
```

Comment:
Are tables created only if there is more than 1 field?

Reply:
yes, I can adjust this if we prefer having a table in this case as well